### PR TITLE
MSI: add missing trigger env var and logs"

### PR DESCRIPTION
**Pull request by rnetser**

### CODE REVIEW

File: ci-operator/step-registry/job-re-trigger/job-re-trigger-commands.sh [hunk 1]
```diff
@@ -20,6 +20,10 @@ if [ "${RE_TRIGGER_ON_FAILURE}" = "true" ]; then
...
 
+  echo "Send job re-triggering for job ${JOB_NAME}, build ${BUILD_ID} prow job ${PROW_JOB_ID}"
+
   curl -X POST  "$SERVER_URL" -d '{"job_name":"'"$JOB_NAME"'", "build_id": "'"$BUILD_ID"'", "prow_job_id":"'"$PROW_JOB_ID"'", "token":  "'"$TOKEN"'"}' -H "Content-Type: application/json"
 
-fi
\ No newline at end of file
+fi
+
+echo "Job was not re-triggered"
```
Comment (myakove) [inline ci-operator/step-registry/job-re-trigger/job-re-trigger-commands.sh:29]: This log is misleading, job was not requested to be re-trigger by the test.

**Review by myakove — CHANGES_REQUESTED** [review]

**Comment by rnetser** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-openshift-ci-trigger-poc-test-fail-setup

**Comment by rnetser** [comment]
/pj-rehearse periodic-ci-CSPI-QE-MSI-openshift-ci-trigger-poc-test-fail-setup

**Review by myakove — APPROVED** [review]

**Pull request merged**